tools/build/src/che-plugin/che-plugins-meta-yaml-generator.ts

```diff
@@ -36,7 +36,7 @@ export class ChePluginsMetaYamlGenerator {
         const metaId = `${publisher}/${name}`;
         const version = splitIds[2];
         let disableLatest: boolean;
-        // disable latest alias if version is not a number (probably like next/nightly/etc.)
+        // disable latest alias if version is not a number (probably like next/next/etc.)
```

Comment:
this comment doesn't make sense after s/nightly/next; please remove the second next :D